### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by adrian2793**
Removed my library

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @adrian2793. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/03/21 16:52:29 Loaded 6768 libraries from DB
Removing AISwitch
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/arduino279/AISwitch not present
Success!
```

**Pull request merged**
